### ISSUE TITLE
chef-run doesn't install chef-client & run the specified resource/recipe in the same run. 

### ISSUE THREAD

**Issue by ericcalabretta** [chef/chef-apply#76]
## Description
When initially using chef-run to install chef-client & run a resource or recipe only chef-client is installed. The resource or recipe is not ran. 

chef-run appears to work, but "test.txt" is not created:

chef-run winrm://35.166.186.157 --user administrator --password 1234567 file 'C:\test.txt'
[✔] Packaging cookbook... done!
[✔] Generating local policyfile... exporting... done!
[✔] Applying file[C:\test.txt] from resource to target.
└── [✔] [35.166.186.157] Successfully converged file[C:\test.txt].

If I log into the system the file is not there, and InSpec also shows no file:

inspec exec -t winrm://35.166.186.157 --user administrator --password 1234567 win_test.rb

Profile: tests from win_test.rb (tests from win_test.rb)
Version: (not specified)
Target:  winrm://administrator@http://35.166.186.157:5985/wsman:3389

  File c:\test.txt
     ×  should exist
     expected File c:\test.txt to exist

Test Summary: 0 successful, 1 failures, 0 skipped

If I execute the same chef-run command again, the file is successfully  created on the second attempt. 

chef-run winrm://35.166.186.157 --user administrator --password 1234567 file 'C:\test.txt'
[✔] Packaging cookbook... done!
[✔] Generating local policyfile... exporting... done!
[✔] Applying file[C:\test.txt] from resource to target.
└── [✔] [35.166.186.157] Successfully converged file[C:\test.txt].

inspec exec -t winrm://35.166.186.157 --user administrator --password 1234567 win_test.rb

 File c:\test.txt
     ✔  should exist

Test Summary: 1 successful, 0 failures, 0 skipped

The same behavior happens for both individual resources and recipes. I have not attempted using a cookbook. 

## Chef Workstation Version
chef-run: 0.2.4
## Platform Version
From MacOS 12.14.3 to a Windows 2016 machine

**Comment by ericcalabretta** [comment]
@jonsmorrow chef-apply is the new repo for the "chef-run" commands? I thought chef-apply was an older unrelated function to chef-run.

**Comment by jonsmorrow** [comment]
@ericcalabretta yes there was an rfc discussion to take over the chef-apply command space for this. I think that may be revisited, but for now the repo name was created to match that decision with the intent to change the command before 1.0. It's very confusing right now.

**Comment by ericcalabretta** [comment]
Makes total sense, It'd be great to improve the UX before 1.0.

**Comment by yangpeng-chn** [comment]
It seems the command chef-run winrm worked and I would like to ask a question that is not directly related to this post, but can any one take a look at what caused the error in the post below. 
`chef-run winrm`, error: `WinRM::WinRMAuthorizationError`
https://discourse.chef.io/t/chef-run-winrm-winrm-winrmauthorizationerror/16239

**Comment by grv87** [comment]
With current chef-run v0.4.6 an example from documentation:
```
chef-run ssh://... directory /tmp/foo --identity-file ...
```
works for me. The directory is created on the first run.

**Comment by ericcalabretta** [comment]
@grv87 Was chef-client already installed on the box or did that same `chef-run` command both install the chef-client & create the directory successfully?
